apps/business_central/exports/journal_entry/tasks.py

```
@@ -68,7 +68,12 @@ def __construct_journal_entry(self, body: JournalEntry, lineitems: List[JournalE
         for lineitem in lineitems:
             for dimension in lineitem.dimensions:
                 dimension['exported_module_id'] = lineitem.id
-
+            
+            account_attribute_type = DestinationAttribute.objects.filter(workspace_id=body.workspace_id, destination_id=lineitem.accounts_payable_account_id).first()
+            balance_account_type_debit_line = 'G/L Account'
+            if account_attribute_type and account_attribute_type.attribute_type == 'BANK_ACCOUNT':
+                balance_account_type = 'Bank Account'
```

Ruff (0.8.0)

Local variable `balance_account_type` is assigned to but never used

Remove assignment to unused variable `balance_account_type`

(F841)